<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UPCClass</class>
 <widget class="QMainWindow" name="UPCClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UPC</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>4</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <widget class="QWidget" name="widget" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>20</number>
           </property>
           <item>
            <widget class="QPushButton" name="btnUPC">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>60</height>
              </size>
             </property>
             <property name="text">
              <string>商品管理</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnPO">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>60</height>
              </size>
             </property>
             <property name="text">
              <string>进货(P.O)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnInventory">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>60</height>
              </size>
             </property>
             <property name="text">
              <string>库存(Inventory)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnOrders">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>60</height>
              </size>
             </property>
             <property name="text">
              <string>订单(Orders)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnSales">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>60</height>
              </size>
             </property>
             <property name="text">
              <string>销量(Sales)</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_6">
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="widget_10" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="btnBack0">
             <property name="text">
              <string>&lt;back</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_8">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="btnNewUPC">
             <property name="text">
              <string>new</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnDeleteUPC">
             <property name="text">
              <string>delete</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineFindUPC"/>
           </item>
           <item>
            <widget class="QPushButton" name="btnFindUPC">
             <property name="text">
              <string>find</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_11" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <property name="topMargin">
            <number>9</number>
           </property>
           <item>
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>类别:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBoxUPC1"/>
           </item>
           <item>
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>公司:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBoxUPC2"/>
           </item>
           <item>
            <spacer name="horizontalSpacer_9">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidgetUPC"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="widget_2" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="btnBack1">
             <property name="text">
              <string>&lt;back</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="btnNewPo">
             <property name="text">
              <string>new</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnDeletePo">
             <property name="text">
              <string>delete</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineFindPo"/>
           </item>
           <item>
            <widget class="QPushButton" name="btnFindPo">
             <property name="text">
              <string>find</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_3" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="topMargin">
            <number>9</number>
           </property>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>类别:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBoxPo1"/>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>公司:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBoxPo2"/>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>购买途径:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBoxPo3"/>
           </item>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>购买时间:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBoxPo4"/>
           </item>
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>运输成本</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBoxPo5"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidgetPo"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_3">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="widget_5" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="btnBack2">
             <property name="text">
              <string>&lt;back</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="btnNewInvent">
             <property name="text">
              <string>new</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnDeleteInvent">
             <property name="text">
              <string>delete</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineFindInvent"/>
           </item>
           <item>
            <widget class="QPushButton" name="btnFindInvent">
             <property name="text">
              <string>find</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_4" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <property name="topMargin">
            <number>9</number>
           </property>
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>类别:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBoxInvent1"/>
           </item>
           <item>
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>公司:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBoxInvent2"/>
           </item>
           <item>
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Total Inventory less than</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineInventLess"/>
           </item>
           <item>
            <widget class="QPushButton" name="btnFindInventLess">
             <property name="text">
              <string>find</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidgetInvent"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_4">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="widget_7" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="btnBack3">
             <property name="text">
              <string>&lt;back</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="btnNewOrder">
             <property name="text">
              <string>new</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnDelOrder">
             <property name="text">
              <string>delete</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineFindOrder"/>
           </item>
           <item>
            <widget class="QPushButton" name="btnFindOrder">
             <property name="text">
              <string>find</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_6" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <property name="topMargin">
            <number>9</number>
           </property>
           <item>
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>类别:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBoxOrders1"/>
           </item>
           <item>
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>公司:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBoxOrders2"/>
           </item>
           <item>
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>是否已经交货</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBoxOrders3"/>
           </item>
           <item>
            <spacer name="horizontalSpacer_6">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_13" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_14">
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_19">
             <property name="text">
              <string>From:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateEdit" name="dateEditOrderFrom">
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_18">
             <property name="text">
              <string>To:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateEdit" name="dateEditOrderTo">
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_11">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidgetOrder"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_5">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="widget_9" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="btnBack4">
             <property name="text">
              <string>&lt;back</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="btnNewSales">
             <property name="text">
              <string>new</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnDelSales">
             <property name="text">
              <string>delete</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineFindSales"/>
           </item>
           <item>
            <widget class="QPushButton" name="btnFindSales">
             <property name="text">
              <string>find</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_8" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <property name="topMargin">
            <number>9</number>
           </property>
           <item>
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>类别:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBoxSales1"/>
           </item>
           <item>
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>公司:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBoxSales2"/>
           </item>
           <item>
            <spacer name="horizontalSpacer_7">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidgetSales"/>
        </item>
        <item>
         <widget class="QWidget" name="widget_12" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_13">
           <item>
            <spacer name="horizontalSpacer_10">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_total">
             <property name="text">
              <string>total: </string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineTotalProfit">
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="upc.qrc"/>
 </resources>
 <connections/>
</ui>
